usecase/usecase-webapi-xwalk-tests/samples/FileSystem/js/main.js

```diff
@@ -120,24 +112,31 @@ function loaded(evt) {
     }
 }
 
-function loadend(evt) {
-    $.mobile.hidePageLoadingMsg();
-}
-
 function errorHandler(evt) {
    $("#messageInfo").addClass("errorMessage");
    $("#messageInfo").html(evt.target.error.name);
 }
 
-function savefile(index){
-   var file = document.getElementById("fileUpload").files[index];
+function savefile(){
+   var bb = new Blob(["This new content comes from FileWriter"], {type: 'text/plain'});
+   var newfile;
    window.requestFileSystem = window.requestFileSystem || window.webkitRequestFileSystem;
    requestFileSystem(TEMPORARY, 1024*1024,
    function(fs) {
-       fs.root.getFile(file.name, {create: true},
+       fs.root.getFile("test.txt", {create: true},
```

Comment:
Can we avoid this hard code? Or put it in a constant variable making it configurable.